### PR TITLE
Add Anthropic converter

### PR DESCRIPTION
**Pull request by anjor**

### CODE REVIEW

**Comment by anjor** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deployradiant/pychatml/16?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #14** <a href="https://app.graphite.dev/github/pr/deployradiant/pychatml/14?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #15** <a href="https://app.graphite.dev/github/pr/deployradiant/pychatml/15?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #16** <a href="https://app.graphite.dev/github/pr/deployradiant/pychatml/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/deployradiant/pychatml/16?utm_source=stack-comment).

File: pychatml/anthropic_converter.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import re
+from typing import List, Dict, Any
+from anthropic import AI_PROMPT, HUMAN_PROMPT
+
+from pychatml.chatml_converter import ChatMLConverter
```
Comment (koogle) [inline pychatml/anthropic_converter.py:5]: I think chatml converter is a weird  name both for the class and the package. Maybe just abstract and abstract converter

File: tests/test_anthropic.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import unittest
 from anthropic import AI_PROMPT, HUMAN_PROMPT
 
-import pychatml
+from pychatml.anthropic_converter import Anthropic
```
Comment (koogle) [inline tests/test_anthropic.py:4]: I can’t decide if having two words in a module is right I don’t think I have it seen anywhere

Comment (anjor) [reply tests/test_anthropic.py:4]: It was to avoid clashing with the existing folders `anthropic` etc.

Comment (anjor) [reply tests/test_anthropic.py:4]: also i have seen it elsewhere -- for example `from openai import openai.openai_object`

**Review by koogle — APPROVED** [review]

File: tests/test_anthropic.py (hunk 2 above)
Comment (nkulkarni) [reply tests/test_anthropic.py:4]: @koogle otherwise we have a name collision which because python is un-resolvable essentially

**Comment by anjor** [comment]
### Merge activity

* **Jan 2, 4:24 AM**: [Graphite](https://app.graphite.dev/github/pr/deployradiant/pychatml/16) rebased this pull request as part of a merge.

**Pull request merged**